checks/runtimeCheck.go

```diff
@@ -23,12 +24,17 @@ func checkAndReturnRuntime() (runtimeConfig, error) {
 }
 
 func latestAgentTag(r *runtimeConfig) error {
-	resp, err := http.Get(r.agentVersionUrl)
+	client := &http.Client{
+		CheckRedirect: func(req *http.Request, via []*http.Request) error {
+			return http.ErrUseLastResponse
+		},
+	}
+	resp, err := client.Get(r.agentVersionUrl)
```

Comment:
By default the go http client doesn't set a timeout. We should set a timeout in the http client config so that this doesn't potentially block indefinitely.